signature/ecc-sign-verify/Makefile

```diff
@@ -0,0 +1,18 @@
+CC=gcc
+CFLAGS=-Wall
+LIBS=
+
+WOLFSSL_DIR=/usr/local/opt
```

Comment:
Why `/usr/local/opt` here? Default is usually /usr/local. You can also use `WOLFSSL_DIR := ` to allow it to be overridden.

Reply:
Done, thank you David.